### PR TITLE
coefficient

### PR DESCRIPTION
**Pull request by yuanqing-wang**

### CODE REVIEW

**Comment by jchodera** [comment]
Should we have @maxentile review this when ready?

File: espaloma/mm/angle.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def harmonic_angle(x, k, eq):
 
     # NOTE:
     # 0.25 because all angles are calculated twice
```
Comment (maxentile) [inline espaloma/mm/angle.py:32]:
```suggestion
    # 0.5 because all angles are calculated twice
```

File: espaloma/mm/bond.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def harmonic_bond(x, k, eq):
 
     # NOTE:
     # 0.25 because all bonds are calculated twice
```
Comment (maxentile) [inline espaloma/mm/bond.py:32]:
```suggestion
    # 0.5 because all bonds are calculated twice
```

**Review by maxentile — APPROVED** [review]
Nice! Looks like the openmm bond and angle consistency checks now pass

**Pull request merged**